warning: format string is not a string literal

CRAN has asked for a fix the following new problem

(from https://cran.r-project.org/web/checks/check_results_later.html)


```
Version: 1.3.1
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    later_posix.cpp:136:14: warning: format string is not a string literal (potentially insecure) [-Wformat-security]
  See ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/later.Rcheck/00install.out’ for details.
  * used C compiler: ‘Debian clang version 17.0.5 (1)’
  * used C++ compiler: ‘Debian clang version 17.0.5 (1)’
Flavor: [r-devel-linux-x86_64-debian-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/later-00check.html)
```

--- a/NEWS.md
+++ b/NEWS.md
@@ -4,6 +4,8 @@
 
 * Fixed #177: The order of includes in `later.h` could cause compilation errors on some platforms. (@jeroen, #178)
 
+* Closed #181: Fix R CMD check warning re error() format strings (for r-devel). (#133)
+
 # later 1.3.1
 
 * For C function declarations that take no parameters, added `void` parameter. (#172)

--- a/src/later_posix.cpp
+++ b/src/later_posix.cpp
@@ -133,7 +133,7 @@ static void async_input_handler(void *data) {
     std::string msg = "later: exception occurred while executing callback: \n";
     msg += e.what();
     msg += "\n";
-    REprintf(msg.c_str());
+    REprintf("%s", msg.c_str());
   }
   catch( ... ){
     REprintf("later: c++ exception (unknown reason) occurred while executing callback.\n");

